uswds/uswds-site
# USWDS-Site - Bump `stylelint` & `@18f/identity-stylelint-config`

# Summary

Bump `stylelint` and `@18f/identity-stylelint-config` to resolve semver vulnerability.

_Note:_ This PR takes a similar approach to USWDS #[5373](https://github.com/uswds/uswds/pull/5373), however, Site did not need to update to `postcss` so it was left out for now

## Breaking change

This is not a breaking change.

## Related issue

Closes Dependabot alert [#49](https://github.com/uswds/uswds-site/security/dependabot/49)

## Related pull requests

Dependabot Security Update #2165 

## Preview link

Preview link:
Site →

## Problem statement

`Stylelint` has vulnerability in semver dependency. 

Updating this dependency alone causes conflicts with `@18f/identity-stylelint-config`

## Solution

Bump `stylelint` and relevant dependencies that are needed in order to prevent errors.

- Bump `stylelint 14.13.0 → 15.10.1`
- Bump `@18f/identity-stylelint-config 1.0.0 → 2.0.0`

## Dependency updates

| Dependency name              | Previous version | New version |
| ---------------------------- | :--------------: | :---------: |
| [stylelint] |     [14.13.0]      |   [15.10.1]   |
| [@18f/identity-stylelint-confige]  |        [ 1.0.0]        |   [2.0.0]   |


## Testing and Review

1. Pull and checkout this branch
2. Run `npm install` and confirm no installation errors
3. Run `npm run lint` 
4. Confirm it runs without error
5. Confirm no files are changed as a result of linting
6. Run `npm run start`
7. Confirm no build errors
8. Confirm no visual regression

### Testing checklist
- [ ] No install errors
- [ ] Stylelint package dependency resolved
- [ ] Linting runs as expected
- [ ] No visual regression

---
